ports/qemu-riscv/entrypoint.s

```diff
@@ -0,0 +1,70 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2023 Alessandro Gatti
```

Comment:
(Very minor!) 2024. In a number of files.

Reply:
The files were created in 2023, so I think they're correct.  Otherwise every year every file in the repo should have its copyright notice updated accordingly, which isn't the case.

Reply:
Yes, if it was written in 2023 then the date can stay as 2023 (it could also be updated to 2023-2024 but that's not critical).